hazelcast/src/main/java/com/hazelcast/instance/impl/DefaultNodeExtension.java

```diff
@@ -274,13 +283,13 @@ private void printNodeInfoInternal(BuildInfo buildInfo, String build) {
     }
 
     protected String getEditionString() {
-        return "Hazelcast Platform";
+        return "Hazelcast";
```

Comment:
I guess we can remove this whole method as it's no longer overridden (i suppose it used to be)